include/configs/crocodile.h

```diff
@@ -69,11 +69,11 @@
 	"get_fit_config=sf probe;" \
 		"mw.l ${loadaddr} 0x0 0x1;" \
 		"sf read ${loadaddr} ${board_id_addr} 0x1;" \
-		"env set fit_config conf-freescale_crocodile-pt1.dtb;" \
+		"env set fit_config conf-freescale_crocodile-rev-b.dtb;" \
 		"if itest.s *${loadaddr} -eq A; then" \
-		"    echo \"Board A - select config conf-freescale_crocodile-pt1.dtb\";" \
+		"    echo \"Board A - select config conf-freescale_crocodile-rev-b.dtb\";" \
```

Comment:
Shall we align also the board `A` letter to the `rev-b`, considering a `rev-b` = `Board B` ?
@js731ca do you know if the electronics sends the board version letter based on the revision letter?

Reply:
As this functionality will be changes anyway by @js731ca, I wouldn't invest too much time into this